pkg/api/api.go

```diff
@@ -70,7 +70,7 @@ func (hs *HTTPServer) registerRoutes() {
 	r.Get("/dashboard/db/:slug", reqSignedIn, redirectFromLegacyDashboardURL, hs.Index)
 	r.Get("/dashboard/script/*", reqSignedIn, hs.Index)
 	r.Get("/dashboard-solo/snapshot/*", hs.Index)
-	r.Get("/d-solo/:uid/:slug", reqSignedIn, hs.Index)
```

Comment:
this is a breaking change (I think, have you tried urls with slug?) try making slug optional

Reply:
@97amarnathk I tried to explain in https://github.com/grafana/grafana/issues/11617#issuecomment-537434630 that you need to add new routes and keep the existing ones (that includes slug-param. So the changes you've made now is what we want, but we still want the existing routes to be kept intact. Does this makes sense to you? Thanks

Reply:
@marefr, So I need to keep the slug URLs too (instead of deleting them)?

Reply:
I have revised the PR.